fido/fido.py

```diff
@@ -10,12 +10,14 @@
 from twisted.internet.defer import CancelledError
 from twisted.internet.defer import Deferred
 from twisted.internet.endpoints import TCP4ClientEndpoint
-from twisted.internet.error import TimeoutError as TwistedTimeoutError
+from twisted.internet.error import ConnectError as TwistedConnectError
```

Comment:
What is the difference between twisted.internet.error.ConnectError and https://github.com/twisted/twisted/blob/8c25f2c86652276474dfdc8fee70f159369b1d13/twisted/internet/error.py#L147 ?